### ISSUE TITLE
No reset style for Input forms in preview

### ISSUE THREAD

**Issue by sedrubal** [FreeCodeCamp/FreeCodeCamp#6941]
### Challenge:

http://www.freecodecamp.com/challenges/create-a-form-element

### Problem:

The preview does not have a reset style for input forms. So on some browsers and systems you can't read the text inside the form.

#### Steps to reproduce:
1. Choose a system and browser with strange settings (e.g. a KDE desktop with dark design and firefox as browser)
2. Look at a:

```html
<input type="text" placeholder="You can't read me">
```

### Screenshots:

![no_reset_stylesheet](https://cloud.githubusercontent.com/assets/5571650/13019681/2ebb672e-d1d1-11e5-89e8-52e9d7f7d948.png)

**Comment by bugron** [comment]
Thanks for a well formatted issue. @FreeCodeCamp/issue-moderators what do you think?

**Comment by AkiraLaine** [comment]
We should have basic styling reset for the input box.

**Comment by sedrubal** [comment]
...radio buttons, checkboxes and buttons are affected, too... But if you don't reset the style, one can learn that this is important :smile:

**Comment by raisedadead** [comment]
@hallaathrad Do you think this is not costly? If not can we tag this as help wanted and add the reset style being discussed?

**Comment by hallaathrad** [comment]
Hm it wouldn't take much effort to rest these... and someone who is using such "dark designs" is probably used to having this sort of broken behaviour in their browsers due to their custom scheme...
They may or not understand their theming is the issue, they'll rather point it out as a fault in our site so might as well fix it. 

I'm in.

**Comment by hallaathrad** [comment]
@sedrubal would you please find links to challenges with these other elements so I can implement a thorough reset? I'm afraid I cannot replicate here by myself.

**Comment by sedrubal** [comment]
Here you are:

- [ ] https://www.freecodecamp.com/challenges/add-a-submit-button-to-a-form
- [ ] https://www.freecodecamp.com/challenges/use-html5-to-require-a-field
- [ ] https://www.freecodecamp.com/challenges/create-a-set-of-radio-buttons
- [ ] https://www.freecodecamp.com/challenges/create-a-set-of-checkboxes
- [ ] https://www.freecodecamp.com/challenges/check-radio-buttons-and-checkboxes-by-default
- [ ] https://www.freecodecamp.com/challenges/nest-many-elements-within-a-single-div-element
- [ ] https://www.freecodecamp.com/challenges/give-a-background-color-to-a-div-element

Thanks for fixing ;)
